modules/claims_api/app/controllers/claims_api/concerns/json_format_validation.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module JsonFormatValidation
+    extend ActiveSupport::Concern
+
+    included do
+      def validate_json_format
+        @json_body = JSON.parse(request.body.string)
+      rescue JSON::ParserError
+        error = {
+          errors: [
+            {
+              type: 'malformed',
+              detail: "The payload body isn't valid JSON format"
```

Comment:
JSON API format?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/3729/commits/304a03c162c82466ad7a15a65afd333168664a94 resolved there